src/IOST/Signer.cpp

```diff
@@ -0,0 +1,131 @@
+#include "Signer.h"
+#include "Account.h"
+#include "../Hash.h"
+#include "../PrivateKey.h"
+#include <boost/endian/conversion.hpp>
+#include <sstream>
+#include <iostream>
+
+#include <nlohmann/json.hpp>
+
+using namespace TW;
+using namespace TW::IOST;
+
+class IOSTEncoder {
+  public:
+    IOSTEncoder() {}
+    void WriteByte(uint8_t b) { buffer << b; }
+    void WriteInt32(uint32_t i) {
+        uint32_t r = boost::endian::endian_reverse(i);
```

Comment:
This will depend on the machine architecture. Please use the functions in `BinaryCoding.h`